lib/components/auth/digest.ts

```diff
@@ -104,10 +104,10 @@ export class DigestAuth {
       this.qop === undefined
         ? new MD5().update(`${ha1}:${this.nonce}:${ha2}`).digest('hex')
         : new MD5()
-            .update(`${ha1}:${this.nonce}:${nc}:${cnonce}:${this.qop}:${ha2}`)
-            .digest('hex')
+          .update(`${ha1}:${this.nonce}:${nc}:${cnonce}:${this.qop}:${ha2}`)
+          .digest('hex')
```

Comment:
We use prettier to handle formatting, but it's not enforce on commit, so we rely on people running it themselves (since I don't like anything changing the code after commit, even a formatter).

You can use e.g. a prettier plugin for Atom or VS Code, or run `yarn prettier path/to/file` to get it working.